### ISSUE TITLE
Provide a link back to the group from the discourse welcome message

### ISSUE THREAD

**Issue by nukeador** [mozilla/community-portal#237]
It would be good if the welcome message created when generating a discourse category for a group, had, after the group description, also a link back to the group. 

This way we can provide context and navigation back to the portal to the people just visiting the discourse site.

For example in this topic:

https://discourse.mozilla.org/t/about-the-mozilla-hispano-category/51609

It should also have at the end a new line with this link: 

https://community.mozilla.org/groups/mozilla-hispano/

**Comment by dnaluz** [comment]
This would be an enhancement to the Discourse code base.  The community portal doesn't make any posts.

**Comment by nukeador** [comment]
But when a group description is changes, this post is also updated, so I suspect this is modified through the API by the portal?

**Comment by nukeador** [comment]
Yeah, maybe we just need to append the link to the description in a new line to solve this.

@LeoMcA

**Comment by LeoMcA** [comment]
The API accepts markdown in the `description` string, and and updates the "about this category" topic in that category.

So appending something like `\n\n[View this group on the Community Portal →](https://community.mozilla.org/groups/blah).` would work.
